book/src/quickstart.md

```diff
@@ -55,13 +55,22 @@ docker run --rm -i -t -v kanidmd:/data \
 docker start kanidmd
 ```
 
-### Recover the admin password
+### Recover the admin roles
```

Comment:
Can we keep the word "password" here so that it's searchable? It's SEO-adjacent ensure that "ugh I need the admin password" thing is findable.

```suggestion
### Recover password for the admin roles
```